### PR TITLE
2140 add active record associations to land

### PR DESCRIPTION
**Pull request by ashzagorski**
Allows ability to call:

```
pageview.events
pageview.events.by_event_type(:eventType)
```

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@
 Combustion.initialize! :active_record,
   database_reset: false,
   load_schema: false,
-  database_migrate: true
+  database_migrate: false
```
Comment (ashzagorski) [inline spec/rails_helper.rb:17]: I was getting an error in CI 
`uninitialized constant ActiveRecord::MigrationContext`
not sure if this was the right solution but everything was passing and CI was happy when I did this

**Comment by ashzagorski** [comment]
[Add ActiveRecord associations to land for easier Pageview access to events](https://trello.com/c/uk58oTa3/2140-add-activerecord-associations-to-land-for-easier-pageview-access-to-events)

File: spec/rails_helper.rb (hunk 1 above)
Comment (jorin) [reply spec/rails_helper.rb:17]: I'm not familiar with `Combustion` ... but [this](https://github.com/pat/combustion#disabling-database-preparation) does lead me to believe that this should be alright.

From the quick read, it seems like it's for testing engines (I guess as opposed to like a dummy app in the spec folder), and that section above makes it sound as though if we're handling db setup as a separate operation, then we would skip these processes.

Which sounds like what we're doing, with having CircleCI build and migrate our postgres test db as an aspect of setting up the runs. So, from my rudimentary understanding scanning those docs that seems entirely cool to me.

**Review by jorin — APPROVED** [review]
+1 super, thanks for doing this. I like the console so this is really nice

**Pull request merged**